ui/appui-layout-react/src/appui-layout-react/preview/PreviewFeatures.tsx

```diff
@@ -18,6 +18,7 @@ export interface KnownPreviewLayoutFeatures {
   contentAlwaysMaxSize: boolean;
   /** If true, the floating widget will have a "maximize" button. */
   enableMaximizedFloatingWidget: boolean;
+  changeActiveTabAfterDragDrop?: boolean;
```

Comment:
Add documentation (I just created the relevant discussion) :
```suggestion
  /** If true, a tab, or the active tab of a group of wiget will become active when dropping them in a container.
   *
   * Discuss or upvote this feature: https://github.com/iTwin/appui/discussions/679
   */
  changeActiveTabAfterDragDrop?: boolean;
```